tfhe/Cargo.toml

```diff
@@ -132,34 +129,15 @@ parallel-wasm-api = ["dep:wasm-bindgen-rayon"]
 
 nightly-avx512 = ["tfhe-fft/nightly", "tfhe-ntt/nightly", "pulp/nightly"]
 
-# Enable the x86_64 specific accelerated implementation of the random generator for the default
-# backend
-generator_x86_64_aesni = ["tfhe-csprng/generator_x86_64_aesni"]
-
-# Enable the aarch64 specific accelerated implementation of the random generator for the default
-# backend
-generator_aarch64_aes = ["tfhe-csprng/generator_aarch64_aes"]
-
 # Private features
 __profiling = []
 __long_run_tests = []
 
-seeder_unix = ["tfhe-csprng/seeder_unix"]
-seeder_x86_64_rdseed = ["tfhe-csprng/seeder_x86_64_rdseed"]
-
-# These target_arch features enable a set of public features for tfhe if users want a known
-# good/working configuration for tfhe.
-# For a target_arch that does not yet have such a feature, one can still enable features manually or
-# create a feature for said target_arch to make its use simpler.
-x86_64 = ["generator_x86_64_aesni", "seeder_x86_64_rdseed"]
-x86_64-unix = ["x86_64", "seeder_unix"]
-
-aarch64 = ["generator_aarch64_aes"]
-aarch64-unix = ["aarch64", "seeder_unix"]
+software_prng = []
```

Comment:
No, we enable it anyways in tfhe-csprng because it is needed as a fallback, for example in wasm. The feature in tfhe is used to "force" the software prng.

The feature in tfhe-csprng could be removed but its mostly to make the panic error message more coherent if the cpu features are not detected at runtime.

Reply:
ok I see